lib/IDE/Refactoring.cpp

```diff
@@ -2965,6 +2965,101 @@ static bool rangeStartMayNeedRename(ResolvedRangeInfo Info) {
   }
   llvm_unreachable("unhandled kind");
 }
+    
+bool RefactoringActionConvertToComputedProperty::
+isApplicable(ResolvedRangeInfo Info, DiagnosticEngine &Diag) {
+
+    if (Info.Kind != RangeKind::SingleDecl)
+        return false;
+
+    if (Info.ContainedNodes.empty())
+        return false;
+
+    if (Info.ContainedNodes.size() == 1) {
```

Comment:
Can you convert these nested `if`s to early returns, like the ones above?

Reply:
Done.

Removed nested ifs and simplify the logic.